arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -11,209 +11,226 @@ import PublicPreviewBannerPartial from '../partials/_public-preview-banner-parti
+`cargo stylus` requires Docker to operate. You can download Docker from [Docker’s website](https://www.docker.com/products/docker-desktop).
 
-If you’re using [MetaMask](https://metamask.io/), simply click the dropdown at the top middle of the plugin and then click “Add Account” to create a fresh account. It can be helpful to label the account as a dev wallet or “Stylus” for this purpose. You’ll need this newly created account’s private key (as well as some Sepolia ETH) for deploying a smart contract. [Follow the instructions on MetaMask’s website](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key) to obtain your key.
+#### Developer wallet/account
 
-![Stylus Wallet](./assets/stylus-wallet.png)
+We recommend creating an account specifically for development that doesn't hold any real assets.
 
-:::caution
+##### Creating a new account
 
-**_Never share your “secret recovery phrase” with anyone_**. Never enter it anywhere. A _private key_ is valid for an individual account only, but a _secret recovery phrase_ can be used to gain access to ALL accounts in your wallet.
+If you’re using [MetaMask](https://metamask.io/), click the dropdown at the top middle of the plugin and then click “Add Account” to create a fresh account.
+Labeling the account as a dev wallet or “Stylus” can be helpful. You’ll need this newly created account’s private key. [Follow the instructions on MetaMask’s website](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key) to obtain your key.
 
-:::
+##### Storing your private key
 
-### Testnet ETH
+You might also want to add your developer's private key to your environment variables for easier and safer access.
+To do this, store your hex-encoded private key in a text file and add the path to the file as an ENV variable named `$PRIVKEY_FILE_PATH` to your shell file, e.g.,``.bashrc`.
 
-The Stylus testnet settles directly to the [Arbitrum Sepolia](/build-decentralized-apps/03-public-chains.md#arbitrum-sepolia) testnet. Follow these steps to acquire testnet ETH on the Stylus testnet:
+#### Arbitrum Sepolia ETH
 
-1. Navigate to [https://bwarelabs.com/faucets/arbitrum-sepolia](https://bwarelabs.com/faucets/arbitrum-sepolia).
-2. Enter your wallet address into the text field.
-3. Click `Claim` and optionally follow the second step to receive extra testnet tokens.
-4. You should now have Sepolia ETH on the Stylus testnet.
+We will interact with the [Arbitrum Sepolia](./reference/testnet-information.md) testnet, so you will need a small amount of Sepolia ETH to deploy contracts and send transactions. 
 
-For additional sources of testnet ETH, please use a faucet on Arbitrum Sepolia or Ethereum Sepolia:
+To add Arb Sepolia ETH to your wallet you can request it from a [faucet](https://arbitrum.faucet.dev/ArbSepolia).
```

Comment:
Maybe better to not directly reference this third party page, but instead the individual faucets. I think you can simply reference the faucets from Alchemy and Quicknode here.

Reply:
Done.